packages/cosmos_ui_components/lib/cosmos_theme.dart

```diff
@@ -87,6 +88,68 @@ class CosmosThemeData extends Equatable {
     this.errorColor = CosmosColorsData.defaultError,
   });
 
+  const CosmosThemeData.dark({
+    this.spacingXXXL = defaultSpacingXXXL,
+    this.spacingXXL = defaultSpacingXXL,
+    this.spacingXL = defaultSpacingXL,
+    this.spacingL = defaultSpacingL,
+    this.spacingM = defaultSpacingM,
+    this.spacingS = defaultSpacingS,
+    this.spacingXS = defaultSpacingXS,
+    this.longDuration = defaultLongDuration,
+    this.mediumDuration = defaultMediumDuration,
+    this.shortDuration = defaultShortDuration,
+    this.radiusXL = defaultRadiusXL,
+    this.radiusL = defaultRadiusL,
+    this.radiusM = defaultRadiusM,
+    this.radiusS = defaultRadiusS,
+    this.fontSizeS = defaultFontSizeS,
+    this.fontSizeM = defaultFontSizeM,
+    this.fontSizeL = defaultFontSizeL,
+    this.fontSizeXL = defaultFontSizeXL,
+    this.fontSizeXXL = defaultFontSizeXXL,
+    this.borderRadiusL = defaultBorderRadiusL,
+    this.borderRadiusM = defaultBorderRadiusM,
+    this.borderRadiusS = defaultBorderRadiusS,
+    this.elevationS = defaultElevationS,
+    this.elevationM = defaultElevationM,
+    this.elevationL = defaultElevationL,
+    this.colors = darkThemeColors,
+    this.brightness = Brightness.dark,
+    this.errorColor = CosmosColorsData.defaultError,
+  });
+
+  const CosmosThemeData.light({
```

Comment:
a lot of redundancy here, you can provide one default constructor with all those default values and reuse that, like so:
```dart
  
const CosmosThemeData({
    required this.colors,
    required this.brightness,
    this.spacingXXXL = defaultSpacingXXXL,
    this.spacingXXL = defaultSpacingXXL,
    this.spacingXL = defaultSpacingXL,
    this.spacingL = defaultSpacingL,
    this.spacingM = defaultSpacingM,
    this.spacingS = defaultSpacingS,
    this.spacingXS = defaultSpacingXS,
    this.longDuration = defaultLongDuration,
    this.mediumDuration = defaultMediumDuration,
    this.shortDuration = defaultShortDuration,
    this.radiusXL = defaultRadiusXL,
    this.radiusL = defaultRadiusL,
    this.radiusM = defaultRadiusM,
    this.radiusS = defaultRadiusS,
    this.fontSizeS = defaultFontSizeS,
    this.fontSizeM = defaultFontSizeM,
    this.fontSizeL = defaultFontSizeL,
    this.fontSizeXL = defaultFontSizeXL,
    this.fontSizeXXL = defaultFontSizeXXL,
    this.borderRadiusL = defaultBorderRadiusL,
    this.borderRadiusM = defaultBorderRadiusM,
    this.borderRadiusS = defaultBorderRadiusS,
    this.elevationS = defaultElevationS,
    this.elevationM = defaultElevationM,
    this.elevationL = defaultElevationL,
    this.errorColor = CosmosColorsData.defaultError,
  });

  const CosmosThemeData.dark() : this(colors: darkThemeColors, brightness: Brightness.dark);
  
  const CosmosThemeData.light() : this(colors: lightThemeColors, brightness: Brightness.light);
 ```